libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTracker.java

```diff
@@ -124,6 +124,7 @@
         NOTIFICATION_UNLIKED,
         NOTIFICATION_TRASHED,
         NOTIFICATION_FLAGGED_AS_SPAM,
+        NOTIFICATION_SWIPE_PAGE_CHANGED,
```

Comment:
Thanks a lot for your explanation and directions on chat! addressed in 9651cdf  - would you take another look please? 🙇